pulpcore.if

```diff
@@ -16,10 +16,30 @@ interface(`pulpcore_filetrans_named_content',`
 	')
 
 	files_pid_filetrans($1, pulpcore_server_var_run_t, { dir }, "pulpcore-content")
-	files_pid_filetrans($1, pulpcore_var_run_t, { dir }, "pulpcore-api")
-	files_pid_filetrans($1, pulpcore_var_run_t, { dir }, "resource-manager")
-	files_pid_filetrans($1, pulpcore_var_run_t, { dir }, "resource-worker-0")
-	files_pid_filetrans($1, pulpcore_var_run_t, { dir }, "resource-worker-1")
+	files_pid_filetrans($1, pulpcore_server_var_run_t, { dir }, "pulpcore-api")
+	files_pid_filetrans($1, pulpcore_var_run_t, { dir }, "pulpcore-resource-manager")
+	files_pid_filetrans($1, pulpcore_var_run_t, { dir }, "pulpcore-worker-1")
+	files_pid_filetrans($1, pulpcore_var_run_t, { dir }, "pulpcore-worker-2")
```

Comment:
It is a systemd service that uses %i and the number of workers is configurable (at least in the Foreman Installer). Is it possible to support an arbitrary number of workers?

Reply:
Same for pulp installer.